keyboards/mechwild/bbs/keyboard.json

```diff
@@ -7,7 +7,19 @@
         "vid": "0x6D77",
         "pid": "0x170E",
         "device_version": "0.0.1",
-        "force_nkro": true
+        "force_nkro": true,
+        "shared_endpoint": {
+            "keyboard": true
+        }
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": false,
+        "dip_switch": true,
+        # Necessary for stenography functionality
```

Comment:
```suggestion
```

Reply:
Do we just not want any of these type of notes present for any board? I kept these because I thought they were useful to have.

Reply:
I was thinking of keymap.json which either Configurator or the compiler didn't like comments in. `format-json` will obliterate them anyway.

Reply:
@fauxpark, when/where does `format-json` run?

@lesshonor, reverting changes to `mechwild/bbs` to defer to your existing PR.

Reply:
s/will/would